protected/humhub/modules/content/notifications/ContentDeleted.php

```diff
@@ -11,9 +11,6 @@
 use humhub\modules\notification\components\BaseNotification;
 use Yii;
 use yii\bootstrap\Html;
-use humhub\modules\user\models\User;
```

Comment:
`$this->originator->displayName` = User?

Reply:
Your ? is the problem in the code. How to we know the relation?